apps/admin-portal/src/components/claims/dynamic-fields.tsx

```diff
@@ -166,8 +170,10 @@ export const DynamicField = (props: DynamicFieldPropsInterface): React.ReactElem
                                                                     if (isSameUserStore) {
                                                                         validation.isValid = false;
                                                                         validation.errorMessages.push(
-                                                                            "This User Store has been selected twice. " +
-                                                                            "A User Store can only be selected once."
+                                                                            (duplicateKeyErrorMsg === undefined) ?
```

Comment:
I think a better way of doing this would be:
```javascript
duplicateKeyErrorMsg ?? "Predefined error message" 
```

Reply:
will change the default error message and override the error message if needed

Reply:
@thivi FYI, nullish coalescing operator doesn't seem to work with the current typescript version we are using.